go.mod

```diff
@@ -0,0 +1,8 @@
+module example.com/m/v2
+
+go 1.20
+
+require (
+	github.com/google/go-querystring v1.1.0
+	github.com/pkg/errors v0.9.1
```

Comment:
Please, use built-in package instead (https://pkg.go.dev/errors), this one is abandoned.